packages/grafana-ui/src/components/Table/TableNG/TableNG.tsx

```diff
@@ -821,6 +822,7 @@ const getGridStyles = (
     border: 'none',
 
     '.rdg-cell': {
+      padding: TABLE.CELL_PADDING,
```

Comment:
a simplification we could have a made a while ago. every header, footer, and normal cell gets 6px padding in every direction, always. you'll see a few other `padding*` declarations get deleted here and there as a result.